content/docs/other-guides/accessing-uis.md

```diff
@@ -17,26 +17,26 @@ Kubeflow comes with a number of web UIs e.g.
 * Katib
 * TFJobs Dashboard
 
-To make it easy to connect to these UIs Kubeflow provides a reverse proxy through
-which all UIs are accessible.
+To make it easy to connect to these UIs Kubeflow provides a left hand navigation
+bar for navigating between the different applications.
 
-Instructions below indicate how to connect to the Kubeflow navigation UI. From
-there you can easily navigate to the different services. The UI looks like this:
+Instructions below indicate how to connect to the Kubeflow landing page. From
+there you can easily navigate to the different services using the left hand navigation
+bar. The landing page looks like this:
 
-![Central UI](/docs/images/central-ui.png)
+![Landing page](/docs/images/landing-page.png)
```

Comment:
This image isn't showing up - it takes up space, but is invisible. See preview:
https://deploy-preview-568--competent-brattain-de2d6d.netlify.com/docs/other-guides/accessing-uis/

Reply:
Fixed.